src/server/pfs/cmds/cmds.go

```diff
@@ -328,7 +328,7 @@ Files can be read from finished commits with get-file.`,
 			if len(args) == 3 {
 				path = args[2]
 			}
-			fileInfos, err := pfsclient.ListFile(apiClient, args[0], args[1], path, "", shard())
+			fileInfos, err := pfsclient.ListFile(apiClient, args[0], args[1], path, "", shard(), true)
```

Comment:
To be clear, setting the `recurse` option to true means that we will correctly compute the sizes of children directories at the cost of performance.  I think this is a reasonable tradeoff for `pachctl` since `pachctl` is meant to be used manually (as opposed to programmatically), and the human user probably expects to see the correct sizes of directories.

This does not match what we had before, since before if you did `list-file`, you'd see child directories' sizes being zero.